### PR TITLE
Few improvements

### PR DESCRIPTION
**Pull request by sakrist**
- Added few optimisations;
- Compatible with Objective-C;
- MacOS framework.

### CODE REVIEW

File: DelaunayTriangulationSwift/Vertex.swift [hunk 1]
```diff
@@ -6,46 +6,45 @@
...
         return result
     }
 }
 
-extension Array where Element: Hashable {
-    func removeDuplicates() -> [Element] {
-        return Array(Set(self))
+extension Vertex: Hashable {
+    public var hashValue: Int {
+        return Int((x+y)*10000000)
```
Comment (AlexLittlejohn) [inline DelaunayTriangulationSwift/Vertex.swift]: This method of hashing is more likely to result in collisions vs the previous method. 1+1 = 2+0 and so on.

File: DelaunayTriangulationSwift/Vertex.swift [hunk 2]
```diff
@@ -6,46 +6,45 @@
...
-}
+extension Vertex: Equatable { }
 
-extension Vertex: Hashable {
-    public var hashValue: Int {
-        return "\(x)\(y)".hashValue
-    }
+public func ==(lhs: Vertex, rhs: Vertex) -> Bool {
+    return lhs.x == rhs.x && lhs.y == rhs.y
 }
```
Comment (AlexLittlejohn) [inline DelaunayTriangulationSwift/Vertex.swift]: This method of implementing the equatable extension feels dirty to me. The implementation is separate from the conformance to the protocol that requires it whereas in the previous code it was together.

File: DelaunayTriangulationSwift/Vertex.swift (hunk 1 above)
Comment (sakrist) [reply DelaunayTriangulationSwift/Vertex.swift]: you right, I did not think about it in this way :) But I think converting to string and then back is inefficient. Will think about it more.

**Comment by sakrist** [comment]
I've replaced hash function with another interesting implementation. 
From here
https://codereview.stackexchange.com/a/148777/28941

Also moved equatable back.

**Pull request merged**